csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -33,7 +33,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" Version="1.0.0"/>
```

Comment:
Why was this removed? It's necessary for the NuGet package to have appropriate SourceLink info.

Reply:
hmm not sure, this was in one of my first commits.  Lemme try putting it back and see if anything breaks